lib/server/Server.js

```diff
@@ -19,6 +20,7 @@ var Server = {
         var onRouteHandled = config.onRouteHandled;
         var ServerAppToUse = config.ServerApp || ServerApp;
         var appRoot = environmentConfig.appRoot;
+        var proxyClientRequestsToApi = config.proxyClientRequestsToApi;
```

Comment:
On second thought, I don't think it would actually break anyone so maybe it could be the default.